#! /bin/sh /usr/share/dpatch/dpatch-run
## 60_php_drop.dpatch by Petter Reinholdtsen <pere@hungry.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to drop PHP support in the package.  Kept here to
## DP: document how the package was changed to drop php mapscript
## DP: support.

@DPATCH@
Index: mapserver/debian/control
===================================================================
RCS file: /cvsroot/pkg-grass/mapserver/debian/control,v
retrieving revision 1.3
diff -u -u -r1.3 control
--- mapserver/debian/control	18 Feb 2005 19:25:23 -0000	1.3
+++ mapserver/debian/control	19 Mar 2005 21:06:19 -0000
@@ -3,19 +3,7 @@
 Priority: optional
 Maintainer: Thomas Sondag <thomas@microdop.org> 
 Standards-Version: 3.6.4
-Build-Depends: debhelper, dpatch, libcurl3-dev, libpng12-dev, zlib1g-dev (>= 1.1.4), libgd2-xpm-dev (>= 2.0.1-10), libfreetype6-dev (>=2.0.9), libjpeg62-dev, libgdal1-dev, proj, libwww-dev, postgresql-dev (>= 7.3.3-1), php4-dev, swig, python-dev
+Build-Depends: debhelper, dpatch, libcurl3-dev, libpng12-dev, zlib1g-dev (>= 1.1.4), libgd2-xpm-dev (>= 2.0.1-10), libfreetype6-dev (>=2.0.9), libjpeg62-dev, libgdal1-dev, proj, libwww-dev, postgresql-dev (>= 7.3.3-1), swig, python-dev
-
-Package: php4-mapscript
-Architecture: any
-Depends: ${shlibs:Depends}, libapache-mod-php4, debconf
-Suggests: mapserver-bin, mapserver-doc
-Replaces: php-mapscript
-Conflicts: php-mapscript
-Description: module for php4-cgi to use mapserver
- php mapscript allows you to use the mapserver functionalities with php web
- scripting language. 
- .
- Curently work only with php as an Apache DSO.
 
 Package: perl-mapscript
 Architecture: any 
@@ -45,7 +33,7 @@
 Package: mapserver-bin 
 Architecture: any
 Depends: ${shlibs:Depends}, ${perl:Depends}
-Suggests: perl-mapscript, python-mapscript, php4-mapscript, cgi-mapserver, mapserver-doc
+Suggests: perl-mapscript, python-mapscript, cgi-mapserver, mapserver-doc
 Description: mapserver binary utilitys
  binary utilitys for using mapserver with the command line.
  .
@@ -66,7 +54,7 @@
 Package: mapserver-doc 
 Architecture: any
 Depends: ${shlibs:Depends}
-Suggests: perl-mapscript, python-mapscript, php4-mapscript, cgi-mapserver, mapserver-doc
+Suggests: perl-mapscript, python-mapscript, cgi-mapserver, mapserver-doc
 Description: documentation for mapserver  
  some short Documentation and examples for mapserver.
  .
Index: mapserver/debian/dirs
===================================================================
RCS file: /cvsroot/pkg-grass/mapserver/debian/dirs,v
retrieving revision 1.1.1.1
diff -u -u -r1.1.1.1 dirs
--- mapserver/debian/dirs	23 Jan 2005 18:15:39 -0000	1.1.1.1
+++ mapserver/debian/dirs	19 Mar 2005 21:06:19 -0000
@@ -1,5 +1,3 @@
 usr/bin
 usr/lib
-usr/lib/php4/20020429/
-usr/lib/perl/5.6.1/auto/mapscript
 usr/include
Index: mapserver/debian/rules
===================================================================
RCS file: /cvsroot/pkg-grass/mapserver/debian/rules,v
retrieving revision 1.6
diff -u -u -r1.6 rules
--- mapserver/debian/rules	19 Mar 2005 21:04:57 -0000	1.6
+++ mapserver/debian/rules	19 Mar 2005 21:06:19 -0000
@@ -113,7 +113,7 @@
 	-$(MAKE) distclean
 
 
-install: install-stamp install-php-stamp
+install: install-stamp # install-php-stamp
 install-stamp: build-stamp
 	dh_testdir
 	dh_testroot
